acme/tests/messages_test.py

```diff
@@ -5,6 +5,7 @@
 import josepy as jose
 
 from acme import challenges
+from acme.magic_typing import Dict
```

Comment:
Why not use `typing` directly, since `magic_typing` is being deprecated? I think new code is already introduced directly into certbot that uses `typing`.

Reply:
Because I created this PR before magic_typing module has been deprecated. I need to update it to remove the remaining legacy code now.

Reply:
:+1: